src/libstd/macros.rs

```diff
@@ -199,6 +225,29 @@ pub mod builtin {
     /// ```rust
     /// use std::fmt;
     ///
+    /// let s = fmt::format(format_arg!("hello"));
+    /// assert_eq!(s, format!("hello"));
+    ///
+    /// ```
+    #[macro_export]
+    macro_rules! format_arg { ($fmt:expr) => ({
```

Comment:
@erickt pointed out my mistake on IRC (interpreted this just a variation of `format_args` rather than a new `format_arg` macro).

In any case, I think the doc string for this needs updating, since AFAICT, it doesn't return `Arguments` any more.